tests/tests_app/core/test_lightning_app.py

```diff
@@ -89,56 +88,6 @@ def run(self):
             self.has_finished = True
 
 
-class SimpleFlow(LightningFlow):
-    def __init__(self):
-        super().__init__()
-        self.work_a = Work(cache_calls=True)
-        self.work_b = Work(cache_calls=False)
-
-    def run(self):
-        self.work_a.run()
-        self.work_b.run()
-        if self.work_a.has_finished and self.work_b.has_finished:
-            self._exit()
-
-
-@pytest.mark.skip
-@pytest.mark.parametrize("component_cls", [SimpleFlow])
-@pytest.mark.parametrize("runtime_cls", [SingleProcessRuntime])
```

Comment:
Probably worth to try out MultiProcessRuntime instead of removing the test.

Reply:
Added back